src/app/receitas/receitas.component.html

```diff
@@ -1 +1,12 @@
-<p>receitas works!</p>
+<div class="main">
+    <div class="modal-header">
+        <h4 class="modal-title">{{ title }}</h4>
+        <a type="button" class="close-btn" aria-label="Close" (click)="activeModal.dismiss('Cross click')">
+            <span aria-hidden="true">×</span>
+        </a>
```

Comment:
Mandou bem adicionando o `aria-label="Close"`, bem importante 👏🏽.

Mas aqui tenho duas considerações:
1. Como esse `x` é um botão de fechar, eu não colocaria `type="button"`, eu já usaria o elemento html `<button>`;
2. Eu acredito que não sejá correto adicionar `aria-hidden="true"` para o `x`, até porque o `x` não estará escontido. Isso pode acabar atrapalhando na acessibilidade.

```suggestion
        <button class="close-btn" aria-label="Close" (click)="activeModal.dismiss('Cross click')">
            <span>×</span>
        </button>
```

Reply:
Ah tranquilo, pode deixar que eu mudo isso.